vizro-core/docs/pages/user-guides/run.md

```diff
@@ -149,6 +149,6 @@ Internally, `app = Vizro()` contains a Flask app in `app.dash.server`. However,
 
 [`Vizro`][vizro.Vizro] accepts `**kwargs` that are passed through to `Dash`. This enables you to configure the underlying Dash app using the same [arguments that are available](https://dash.plotly.com/reference#dash.dash) in `Dash`. For example, in a deployment context, some of the below arguments may be useful:
 
-- `url_base_pathname`: serve your Vizro app at a specific path rather than at the domain root. For example, if you host your dashboard at http://www.example.com/my_dashboard/ then you would set `url_base_pathname="/my_dashboard/"` .
+- `url_base_pathname`: serve your Vizro app at a specific path rather than at the domain root. For example, if you host your dashboard at http://www.example.com/my_dashboard/ then you would set `url_base_pathname="/my_dashboard/"`. Alternatively, you can set the environment variable `DASH_URL_BASE_PATHNAME`.
```

Comment:
I know Vale will suggest using "or" rather than "alternatively". Maybe we could say, "If you prefer, you can set the environment variable..." (also, how do you set it? Could you include the full text, as I assume it needs a path. Is it `DASH_URL_BASE_PATHNAME ="/my_dashboard/"` or ??)